pygsti/objects/operation.py

```diff
@@ -430,8 +430,10 @@ class LinearOperator(_modelmember.ModelMember):
 
     def __init__(self, rep, evotype):
         """ Initialize a new LinearOperator """
-        if isinstance(rep, int):  # For operators that have no representation themselves (term ops)
-            dim = rep             # allow passing an integer as `rep`.
+        # For operators that have no representation themselves (term ops)
+        # allow passing an integer as `rep`.
+        if isinstance(rep, int) or isinstance(rep, _np.int64) or rep == _np.inf:   
```

Comment:
Why do we need _np.inf here?

Reply:
This is a workaround for a bug when creating models with 100s of qubits. I think somewhere in StateSpaceLabels, int overflows are handled by setting the dim to np.inf.

I've removed the change here to make merge with operation.py easy, and will probably start a CHP-specific branch where I start to work around these sorts of issues in less hacky way.